smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/SymbolVisitor.java

```diff
@@ -348,32 +341,25 @@ private ShapeId shadeShapeId(ToShapeId id) {
 
     private Symbol.Builder createGeneratedSymbolBuilder(Shape shape, String typeName, String namespace) {
         return createSymbolBuilder(shape, typeName, namespace)
-                .definitionFile(toFilename(shape, typeName, namespace));
-    }
-
-    private String formatModuleName(ToShapeId id) {
-        StringBuilder result = new StringBuilder();
-
-        for (String part : SHAPE_ID_NAMESPACE_SPLITTER.split(id.toShapeId().getNamespace())) {
-            String[] inner = SHAPE_ID_NAMESPACE_PART_SPLITTER.split(part);
-            for (int i = 0; i < inner.length; i++) {
-                String innerValue = inner[i].toLowerCase(Locale.ENGLISH);
-                if (i > 0) {
-                    innerValue = StringUtils.capitalize(innerValue);
-                }
-                result.append(innerValue);
-            }
-            result.append("/");
-        }
-
-        return result.append("index").toString();
+                .definitionFile(toFilename(namespace));
     }
 
-    private String toFilename(Shape shape, String name, String namespace) {
-        if (shape.isServiceShape()) {
-            return name + "Client.ts";
+    private String formatModuleName(ShapeType shapeType, String name) {
+        // All shapes except for the service and operations are stored in models.
+        if (shapeType == ShapeType.SERVICE) {
+            return "./" + name + "Client";
+        } else if (shapeType == ShapeType.OPERATION) {
+            return "./commands/" + name + "Command";
```

Comment:
"Command" already appended above, resulting in "CommandCommand":

https://github.com/awslabs/smithy-typescript/pull/10/files#diff-dc29ecfad255f6c78afc2c403b56f177R225

Reply:
Fixed and added test